testing/dart/serial_gc_test.dart

```diff
@@ -0,0 +1,17 @@
+// Copyright 2022 The Flutter Authors. All rights reserved.
```

Comment:
Flutter keeps 2013 as the copyright year for all files.

```suggestion
// Copyright 2013 The Flutter Authors. All rights reserved.
```

Reply:
Done.